### PR TITLE
feature: add --quayRoot option (#3)

### PR DESCRIPTION
**Pull request by pbertera**
This PR is the rebase of #3
Unfortunately in the meantime I removed my fork and local branch, so I had to create a new PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ The following flags are also available:
 --quayHostname          The value to set SERVER_HOSTNAME in the Quay config.yaml. This defaults to <targetHostname>:8443
 --initPassword          The password of the init user created during Quay installation.
 --sslCert               The path to the SSL certificate Quay should use
+--sslCheckSkip          Whether or not to check the certificate hostname against the SERVER_HOSTNAME in config.yaml.
```
Comment (HammerMeetNail) [inline README.md:33]: Good catch, thanks for adding this to the docs.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ The following flags are also available:
...
 --sslKey                The path to the SSL key
+--quayRoot          -r  The folder where quay persistent data are saved. This defaults to /etc/quay-install
```
[the remark this answers is not in the document]
Comment (pbertera) [reply README.md:35]: Here https://github.com/quay/openshift-mirror-registry/pull/3#issuecomment-996178232 I understood that `quayRoot` was fine

File: cmd/install.go [hunk 3]
```diff
@@ -80,6 +83,7 @@ func init() {
 
 	installCmd.Flags().StringVarP(&imageArchivePath, "image-archive", "i", "", "An archive containing images")
 	installCmd.Flags().BoolVarP(&askBecomePass, "askBecomePass", "", false, "Whether or not to ask for sudo password during SSH connection.")
+	installCmd.Flags().StringVarP(&quayRoot, "quayRoot", "r", "", "The folder where quay persistent data are saved. This defaults to /etc/quay-install")
```
Comment (HammerMeetNail) [inline cmd/install.go:86]: Can this be updated so that `/etc/quay-install` is the default?

File: cmd/uninstall.go [hunk 4]
```diff
@@ -27,6 +27,7 @@ func init() {
 	uninstallCmd.Flags().StringVarP(&targetHostname, "targetHostname", "H", "localhost", "The hostname of the target you wish to install Quay to. This defaults to localhost")
 	uninstallCmd.Flags().StringVarP(&targetUsername, "targetUsername", "u", os.Getenv("USER"), "The user you wish to ssh into your remote with. This defaults to the current username")
 	uninstallCmd.Flags().BoolVarP(&askBecomePass, "askBecomePass", "", false, "Whether or not to ask for sudo password during SSH connection.")
+	uninstallCmd.Flags().StringVarP(&quayRoot, "quayRoot", "r", "", "The folder where quay persistent data are saved. This defaults to /etc/quay-install")
```
Comment (HammerMeetNail) [inline cmd/uninstall.go:30]: Similar request here, can this also be updated to include the default?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (HammerMeetNail) [reply README.md:35]: Oh, sorry, I must've read that too quickly. Nevermind!

File: cmd/install.go (hunk 3 above)
Comment (pbertera) [reply cmd/install.go:86]: yes, good point I'm fixing that

**Comment by pbertera** [comment]
Latest commit has the default set into the `installCmd.Flags`

**Comment by HammerMeetNail** [comment]
LGTM 👍

**Review by HammerMeetNail — APPROVED** [review]
LGTM 👍

**Pull request merged**